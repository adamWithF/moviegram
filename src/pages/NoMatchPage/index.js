import React from "react"

const NoMatchPage = () => {
  return <div className="no-match-page page">404</div>
}

export default NoMatchPage
