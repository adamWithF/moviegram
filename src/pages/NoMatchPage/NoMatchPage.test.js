import React from "react"
import { shallow } from "enzyme"

import NoMatchPage from "../NoMatchPage"

describe("NoMatchPage", () => {
  it("should render properly", () => {
    const wrapper = shallow(<NoMatchPage />)
    expect(wrapper).toMatchSnapshot()
  })
})
