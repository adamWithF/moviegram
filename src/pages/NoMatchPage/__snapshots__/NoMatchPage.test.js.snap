// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatchPage should render properly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NoMatchPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "404",
      "className": "no-match-page page",
    },
    "ref": null,
    "rendered": "404",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "404",
        "className": "no-match-page page",
      },
      "ref": null,
      "rendered": "404",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
